### PR TITLE
feat: improve about page

### LINKED ISSUE #13: Improve the About Page Layout
At

### PR DESCRIPTION
**Pull request by luhc228**
###  Fixed #13  

before
image
after
image

image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 source "https://rubygems.org"
 gemspec
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```
Comment (huangyz0918) [inline Gemfile:5]: What’s this?

File: _layouts/about.html [hunk 2]
```diff
@@ -4,30 +4,28 @@
...
-
-  {%- if site.author -%}
-    <div class="about-your-name">
-    {{ site.author | escape }}
-    </div>
-  {%- endif -%}
-
-  {%- if site.moving.about_you -%}
-    <p class="about-info">
+  <div class="header">
```
Comment (huangyz0918) [inline _layouts/about.html]: Please don’t format code like this.

File: _sass/moving/_layout.scss [hunk 3]
```diff
@@ -308,34 +308,39 @@
...
   display: inline-block;
-}
 
-.about-your-name {
-  display: flex;
-  flex-direction: row;
-  margin-top: 1.0rem;
-  @include relative-font-size(1.5);
-}
+  .header {
```
Comment (huangyz0918) [inline _sass/moving/_layout.scss]: Instead of header, I think ‘about-header’ will be better.

**Review by huangyz0918 — APPROVED** [review]
Great job @luhc228, I added some suggestions about code style.

File: Gemfile (hunk 1 above)
Comment (luhc228) [reply Gemfile:5]: In order to adapter the windows.

File: _layouts/about.html (hunk 2 above)
Comment (luhc228) [reply _layouts/about.html]: I don't quite understand what you're saying. Could you show me an example?

Comment (luhc228) [reply _layouts/about.html]: > Please don’t format code like this.

I don't quite understand what you're saying. Could you show me an example?

Comment (huangyz0918) [reply _layouts/about.html]: Just follow the original.

File: _sass/moving/_layout.scss (hunk 3 above)
Comment (luhc228) [reply _sass/moving/_layout.scss]: fix

File: _layouts/about.html (hunk 2 above)
Comment (huangyz0918) [reply _layouts/about.html]: Anyway, let me do this code format.

**Pull request merged**